_config.yml

```diff
@@ -16,6 +16,7 @@ gems: [jekyll-paginate]
 main_site_url: https://bazel.build
 blog_site_url: https://blog.bazel.build
 docs_site_url: https://docs.bazel.build
```

Comment:
nit: do we still need `docs_site_url`?

Reply:
Yes, we do. It is used in a lot of old blog posts, example:
```
_posts/2016-06-10-ide-support.md:The [documentation on aspects]({{ site.docs_site_url }}/skylark/aspects.html) explains this in more
```

I don't want to rewrite every single instance of this.